README.md

```diff
@@ -4,13 +4,24 @@ RP Tree is a command-line tool to generate a directory tree diagram.
 
 ## Run the App
 
-Create a Python virtual environment and run the application:
+To run **RP Tree**, you need to download the source code. Then open a terminal or command-line window and run the following steps:
+
+1. Create and activate a Python virtual environment
 
 ```sh
-$ python3 tree.py /path/to/directory/
+$ cd rptree/
+$ python -m venv ./venv
+$ source venv/bin/activate
+(venv) $
 ```
 
-**Note:** The `-h` option provides help on how to use RP Tree.
+2. Run the application
+
+```sh
+(venv) $ python tree.py /path/to/directory/
```

Comment:
Do we want to keep the module name as `tree` or change it to `rptree`?

Would be kind of nice to (eventually) just do a `pip install rptree` and then calling it via `$ rptree ...`

Reply:
That's the idea. I think I'll remove this script and create a `__main__.py` module in `rptree` to provide the entry point script.